### ISSUE TITLE
YT crashes

### ISSUE THREAD

**Issue by dayanch96** [BandarHL/YTShortsDownloader#8]
After building in Xcode and injecting to YouTube, app crashes at start

**Comment by BandarHL** [comment]
Yes there is a lot of problems with this tweak that need to fix and update, but really I don’t have much time for it. I’m currently busy with other important projects.

**Comment by dayanch96** [comment]
Wish you success in your projects. Is there any working compiled dylibs to use?

**Comment by BandarHL** [comment]
The problem is my tweak isn’t compatible with lasted updates of YouTube, even if i give you compiled version it won’t work.

**Comment by dayanch96** [comment]
I thought it was MonkeyDev issues, same was with BHTwitter. I’ll check it again. Thanks for letting know

**Comment by samir0509090** [comment]
💔💔
